default.xml

```diff
@@ -14,4 +14,11 @@
 	<project path="multiap/agent" name="prplMesh-agent" />
 	<project path="multiap/tools" name="prplMesh-tools" />
 
+	<!-- upstream nng -->
+	<remote name="nng" fetch="ssh://git@github.com/nanomsg" />
```

Comment:
please move the remote definition up, next to the other remotes and before the default entry